server/src/test/resources/indices/bwc/testIndex-6.3.0.md

```diff
@@ -0,0 +1,57 @@
+# README for _testIndex-6.3.0.zip_
```

Comment:
Just an organizational nitpick, but would it make sense to created an `es-6.3.0` (or similarly named) subfolder here? Then this file could just be README.md and refer to all files in its folder, though there is only one at this point.

Also I'd put "es" or something similar in the file name. This has the potential to be around for a long long time and will eventually conflict with an OpenSearch 6.x version.